Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -1495,6 +1473,39 @@ bool HGCalDDDConstants::waferInLayerTest(int wafer, int lay, bool full) const {
   return in;
 }
 
+std::pair<double, double> HGCalDDDConstants::waferPosition(int lay,
+                                                           int waferU,
+                                                           int waferV,
+                                                           bool reco,
+                                                           bool
+#ifdef EDM_ML_DEBUG
```

Comment:
is this really necessary?

Reply:
it looks a similar situation to another update of a few days ago, where we preferred at the end not to have the argument list conditional to a debug flag

Reply:
Since debug is enclosed within ifdef, clang complains of unused variable

Reply:
Is the debug flag really needed in addition to the ifdef?

Reply:
I can remove ifdef part for this code. "debug" is needed for some special debugging